test/unit/app/browser/reducers/ledgerReducerTest.js

```diff
@@ -870,19 +871,29 @@ describe('ledgerReducer unit tests', function () {
   })
 
   describe('APP_ON_LEDGER_FUZZING', function () {
```

Comment:
(nit) could you add a case for `newStamp: 0`? Then coverage should be at 100% :D

Reply:
test added